primer.md

```diff
@@ -140,6 +140,7 @@ The following are considered beyond the scope of the specification:
 - Selecting a single identity/access control system
 - Inclusion of protocol-level routing information
 - Event persistence processes
+- Vendor specific Authorization principles
```

Comment:
I would suggest that vendor specificity is less direct than the concerns of [authorization](#378), [integrity](#143), and [confidentiality](#379).  I would recommend directly identifying those concerns.  Consider that open source standards can be unattached to a vendor but that we are not now declaring a correct mechanism for satisfying those requirements.

As a side note, recall that we support satisfying those concerns in more standardized forms through informal extensions or in individualized forms through the data payload.

Side note: the potential mention I declared during the call was tied up in the persistence comment and did not directly address this concern.  Apologies for that noise.

Reply:
I've rephrased the entire thing. please have a look.